subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -131,6 +131,18 @@ uint8_t ll_scan_enable(uint8_t enable)
 		return BT_HCI_ERR_CMD_DISALLOWED;
 	}
 
+	scan->lll.period = (uint64_t)period * (1280000U / 625U);
+	scan->lll.duration = (uint64_t)duration * (10000U / 625U);
```

Comment:
Store in `interval` units and use countdown

Reply:
Fixed